cockatrice/src/dlg_settings.cpp

```diff
@@ -511,7 +529,7 @@ DeckEditorSettingsPage::DeckEditorSettingsPage()
     //priceTagsCheckBox.setChecked(settingsCache->getPriceTagFeature());
     //connect(&priceTagsCheckBox, SIGNAL(stateChanged(int)), settingsCache, SLOT(setPriceTagFeature(int)));
 
-    connect(this, SIGNAL(priceTagSourceChanged(int)), settingsCache, SLOT(setPriceTagSource(int)));
+    //connect(this, SIGNAL(priceTagSourceChanged(int)), settingsCache, SLOT(setPriceTagSource(int)));
```

Comment:
Just a note on this; This was popping up as a run time error. This should've been removed in the past when we removed pricing data.